src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -613,6 +620,33 @@ private void scrape(List<MetricFamilySamples> mfs) throws CloneNotSupportedExcep
         for (Map.Entry<String, ArrayList<MetricFamilySamples.Sample>> entry : extendedSamples.entrySet()) {
           mfs.add(new MetricFamilySamples(baseName + "_" + safeName(toSnakeCase(entry.getKey())), Type.GAUGE, help(rule, unit, entry.getKey()), entry.getValue()));
         }
+        
+        // Add the "aws_resource_info" metric for existing tag mappings
+        for (ResourceTagMapping resourceTagMapping : resourceTagMappings) {
+          if (!publishedResourceInfo.contains(resourceTagMapping.getResourceARN())) {
+            List<String> labelNames = new ArrayList<String>();
+            List<String> labelValues = new ArrayList<String>();
+            labelNames.add("job");
+            labelValues.add(jobName);
+            labelNames.add("instance");
+            labelValues.add("");
```

Comment:
why is there this empty string here ?

Reply:
I am not sure, but I want to stay aligned with other metrics that are also including an empty `instance`. I will verify this while opening the PR to the upstream project.